lib/elixir/lib/calendar/datetime.ex

```diff
@@ -364,8 +365,9 @@ defmodule DateTime do
       {:ok, ~U[2016-05-24 13:26:08.003Z]}
 
   When the datetime is ambiguous - for instance during changing from summer
-  to winter time - the two possible valid datetimes are returned. First the one
-  that happens first, then the one that happens after.
+  to winter time - the two possible valid datetimes are returned in a tuple.
+  Its first element is the datetime that takes place first, and its second element
+  the one that takes place after.
```

Comment:
This is ambiguous. The first element of the tuple is an atom! :)

```suggestion
  The first datetime is also the one who comes first chronologically, while the
  second element comes last.
```

Reply:
True, but still we cannot say "second element", we need to say "while the second datetime comes last"

Reply:
or "the second one comes last"

Reply:
Yup, updated.